go-controller/pkg/ovn/default_network_controller.go

```diff
@@ -770,12 +773,22 @@ func (h *defaultNetworkControllerEventHandler) AddResource(obj interface{}, from
 			if err = h.oc.addUpdateLocalNodeEvent(node, nodeParams); err != nil {
 				klog.Infof("Node add failed for %s, will try again later: %v",
 					node.Name, err)
-				return err
+				errs = append(errs, err)
```

Comment:
if we fail in addUpdateLocalNodeEvent, do we still want to try to add the nodeIP to the address-set? maybe answer is yes but hoping this was consciously taken.? We can choose to go the other way which is to return the error immediately at each stage and then get retried ?